### DISCUSSION TITLE
Bar To fill up via spaming

### DISCUSSION THREAD

**Discussion by ghost** [darmiel/twitch-poll-overlay#17] [Ideas]
Hello Darmiel,

Loving your work with the twitch poll overlay - looking to make a bar that fills up with the chat spamming a word/number/phase/emote and wondered if this is something you have done before.... 

Many thanks,

Ben

**Comment by darmiel** [comment]
Hi Ben,

I'm glad you like the little project.

Do you mean a kind of "hype" bar, so that e.g. 100x the word "yes" should be reached, and then the bar fills up?
